documentation/modules/ROOT/pages/connectors/oracle.adoc

```diff
@@ -37,6 +37,9 @@ For information about the Oracle Database versions that are compatible with this
 endif::product[]
 
 {prodname} ingests change events from Oracle by using the native LogMiner database package
+ifdef::product[]
+or the https://docs.oracle.com/database/121/XSTRM/xstrm_intro.htm#XSTRM72647[XStream API].
```

Comment:
```suggestion
or the https://docs.oracle.com/en/database/oracle/oracle-database/19/xstrm/introduction-to-xstream.html[XStream API].
```

We should probably update these links to Oracle to be 19.x compliant rather than based on 12.x since we no longer officially support Oracle 12 in Debezium 3.0.